.github/workflows/docs.yaml

```diff
@@ -26,16 +26,14 @@ jobs:
           python-version: '3.12'
       - name: Update pip
         run: python -m pip install --upgrade pip
-      - name: Install dependencies and FFmpeg
+      - name: Install torchcodec from nightly
         run: |
-          # TODO: torchvision and torchaudio shouldn't be needed. They were only added
-          #  to silence an error as seen in https://github.com/pytorch/torchcodec/issues/203
-          python -m pip install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/cpu
-          conda install "ffmpeg=7.0.1" pkg-config -c conda-forge
-          ffmpeg -version
-      - name: Build and install torchcodec
+          pip3 install --pre torch torchvision torchaudio torchcodec --index-url https://download.pytorch.org/whl/nightly/cu124
+      - name: Install FFMPEG and other deps
```

Comment:
Before committing this, can  you double check that the doc build can pick up a change that we make to the code? e.g. modify a docstring within the repo, and maybe `print()` something within the decoder, and make sure these 2 changes are reflected within the doc build.
I *think* it will work but I'm not 100% sure.

Reply:
I can test but I don't think it will work because this is picking up the nightly from download.pytorch.org